wire-compiler/src/test/java/com/squareup/wire/logger/MockWireLogger.java

```diff
@@ -0,0 +1,16 @@
+package com.squareup.wire.logger;
+
+import com.squareup.wire.OutputArtifact;
```

Comment:
This class isn't necessary - I could have substituted StringWireLogger in the one case where I decided to use MockWireLogger.  Or we could pull in a mocking library like mockito.